WakaTime/WakaTimeRelease.entitlements

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Is this file used?

Reply:
There's `WakaTime.entitlements` so I believe it was created automatically

Reply:
But we don't have any `WakaTimeRelease.app` so let's try deleting this file?